### PR TITLE
BBFR103/RX888  detection patch

### PR DESCRIPTION
**Pull request by ik1xpv**
GPIO clock init before HWconfig detection
It enables the GPIO logic before use of GPIO to detect hardware then initialize GPIOs for HWconfig detected
It solves https://github.com/ik1xpv/ExtIO_sddc/pull/41#issuecomment-737273416 too.

### CODE REVIEW

File: SDDC_FX3/RunApplication.c [hunk 1]
```diff
@@ -170,6 +157,9 @@ void ApplicationThread ( uint32_t input)
 
     CyU3PReturnStatus_t Status;
     HWconfig = 0;
+
+    HF103_GpioInit();
```
Comment (howard0su) [inline SDDC_FX3/RunApplication.c:161]: can we rename this function?

**Review by howard0su — APPROVED** [review]

File: SDDC_FX3/RunApplication.c (hunk 1 above)
Comment (ik1xpv) [reply SDDC_FX3/RunApplication.c:161]: I propose GpioInitClock()  ?

Comment (howard0su) [reply SDDC_FX3/RunApplication.c:161]: sounds good.

**Comment by howard0su** [comment]
I verified this works well with mk2

**Pull request merged**